src/Raven.Client/Documents/Changes/ChangeNotification.cs

```diff
@@ -256,19 +257,77 @@ public class IndexRenameChange : IndexChange
         /// </summary>
         public string OldIndexName { get; set; }
     }
-
-    public class TrafficWatchChange : DatabaseChange
+    internal abstract class TrafficWatchChangeBase : DatabaseChange, IDynamicJson
     {
+        public abstract TrafficWatchType TrafficWatchType { get; }
         public DateTime TimeStamp { get; set; }
+        public string DatabaseName { get; set; }
+        public string CustomInfo { get; set; }
+        public string ClientIP { get; set; }
+        public string Thumbprint { get; set; }
```

Comment:
CertificateThumbprint - it needs to be clear what thumbprint it is

Reply:
@shiranshalom after you fix server side, please assign issue to me - I will refactor studio part.